lib/pengine/unpack.c

```diff
@@ -1223,7 +1229,9 @@ unpack_remote_status(xmlNode * status, pe_working_set_t * data_set)
             this_node->details->standby = TRUE;
         }
 
-        if (crm_is_true(g_hash_table_lookup(this_node->details->attrs, "maintenance"))) {
+        if (crm_is_true(g_hash_table_lookup(this_node->details->attrs, "maintenance")) ||
+            (this_node->details->remote_rsc &&
+             !is_set(this_node->details->remote_rsc->flags, pe_rsc_managed))) {
```

Comment:
I noticed remote_rsc is used above this for the shutdown case, so it's probably worthwhile to pull that rsc variable to a wider scope and use it here. I also noticed the shutdown case doesn't ensure rsc is not NULL, it should (since we haven't seen any problems reported about that, I'm not sure if it's impossible to get here with a NULL rsc, or just unlikely in practice -- but either way, it doesn't hurt to be sure).